File: C++/M01_ProgrammingBasics/L02_ConditionalStatements/Exercises/Solutions/P03_SpeedInfo.cpp
#include <iostream>
using namespace std;

int main() {

	double speed;
	cin >> speed;

	string output;
	if (speed <= 10) {
		output = "slow";
	} else if (10 < speed && speed <= 50) {
		output = "average";
	} else if (50 < speed && speed <= 150) {
		output = "fast";
	} else if (150 < speed && speed <= 1000) {
		output = "ultra fast";
	} else if (1000 < speed) {
		output = "extremely fast";
	}

	cout << output << endl;

	return 0;
}

File: Python/M01_ProgrammingBasics/L02_ConditionalStatements/Exercises/Solutions/P03_SpeedInfo.py
speed = float(input())

if speed <= 10:
    print("slow")
else:
    if speed <= 50:
        print("average")
    else:
        if speed <= 150:
            print("fast")
        else:
            if speed <= 1000:
                print("ultra fast")
            else:
                print("extremely fast")